### ISSUE TITLE
Request Installation Guide For Denariiumx-server

### ISSUE THREAD

**Issue by knkrth** [carsenk/denariiumx-server#2]
Hi, Can you provide me a installation guide on ubuntu because from https://github.com/kyuupichan/electrumx/blob/master/docs/HOWTO.rst doesn't work for me nor I don't quit get it.

**Comment by metaspartan** [comment]
Yes that is the installation guide, its fairly straight forward, just Python3.6, denariusd, and some configuration, though Denariium is still a WIP and may not work properly yet, upon release, binaries will be provided for each OS. Compiling or installing from the git currently without development experience with Electrum or Python is not recommended and not supported. If you could explain to me what you do not get about it, I could help you a bit further ;)

**Comment by knkrth** [comment]
I Managed to start the server but it stocks on this,

`INFO:root:ElectrumX server starting
WARNING:Env:lowered maximum sessions from 1,000 to 674 because your open file limit is 1,024
INFO:Controller:event loop policy: None
INFO:FakeEstimateFeeDaemon:daemon #1 at 127.0.0.1:33339/ (current)
INFO:BlockProcessor:switching current directory to /home/ubuntu/.electrumx/
INFO:BlockProcessor:using leveldb for DB backend
INFO:BlockProcessor:opened DB for serving
INFO:BlockProcessor:closing DB to re-open for sync
INFO:BlockProcessor:opened DB for sync
INFO:BlockProcessor:software version: ElectrumX 1.1
INFO:BlockProcessor:DB version: 5
INFO:BlockProcessor:coin: Denarius
INFO:BlockProcessor:network: mainnet
INFO:BlockProcessor:height: -1
INFO:BlockProcessor:tip: 0000000000000000000000000000000000000000000000000000000000000000
INFO:BlockProcessor:tx count: 0
INFO:BlockProcessor:flush count: 0
INFO:BlockProcessor:sync time so far: 00s
INFO:BlockProcessor:reorg limit is 1,000 blocks
INFO:BlockProcessor:flushing DB cache at 1,200 MB
INFO:Controller:RPC server listening on localhost:8000
ERROR:FakeEstimateFeeDaemon:disconnected.  Retrying occasionally...`

**Comment by metaspartan** [comment]
Yes it's still a WIP, the FakeFeeEstimateDaemon can be specified in the lib/coins.py, I need to get it corrected for DNR, if you end up finding a fix before me, feel free to put up a PR.
